h2o-algos/src/main/java/hex/glm/ComputationState.java

```diff
@@ -67,7 +67,8 @@ public ComputationState(Job job, GLMParameters parms, DataInfo dinfo, BetaConstr
     _dinfo = dinfo;
     _activeData = _dinfo;
     _intercept = _parms._intercept;
-    _nclasses = (parms._family == Family.multinomial||parms._family == Family.ordinal)?nclasses:1;
+    _nclasses = parms._family.equals(Family.fractionalbinomial)?2:
+            ((parms._family == Family.multinomial||parms._family == Family.ordinal)?nclasses:1);
```

Comment:
```suggestion
    _nclasses = Family.fractionalbinomial == parms._family ? 2 :
            ((Family.multinomial == parms._family || Family.ordinal == parms._family) ? nclasses : 1);
```

Formatting only and slight change of syntax for improved readability.